test/screenshot/infra/lib/selenium-api.js

```diff
@@ -982,16 +982,17 @@ class SeleniumApi {
    * @private
    */
   isBusinessHours_() {
-    const MORNING_UTC_HOURS = 13; // 1pm UTC === 9am EST
-    const EVENING_UTC_HOURS = 1; // 1am UTC === 6pm PST
-    const SATURDAY = 7;
+    const MORNING_HOUR = 9; // 9am
```

Comment:
I think you just changed this from 9 ET to 9 PT

Reply:
Fixed